includes/class-base-widget.php

```diff
@@ -312,6 +312,12 @@ protected function before_form_field( array $field ) {
 	 */
 	protected function render_form_input( array $field ) {
 
+		if ( ! is_admin() ) {
```

Comment:
@jonathanbardo Whoops, this was left over from some testing I was doing. I was getting the 1 value from the checkbox field printing on the front end of the site, and I was trying to hunt it down.